corehq/apps/registration/static/registration/js/new_user.ko.js

```diff
@@ -38,11 +38,14 @@ hqDefine('registration/js/new_user.ko', function () {
 
     // Can't set up analytics until the values for the A/B tests are ready
     _kissmetrics.whenReadyAlways(function() {
-        _private.isAbPhoneNumber = _kissmetrics.getAbTest('New User Phone Number') === 'show_number';
+        _private.isAbPhoneNumberShown = _kissmetrics.getAbTest('New User Phone Number') === 'show_number';
+        _kissmetrics.track.event("Viewed CommCare signup page", {
```

Comment:
This event doesn't need the data, either.

Reply:
ok, so `identifyTraits` it is